app/src/main/java/com/example/taskyy/ui/screens/ReminderScreen.kt

```diff
@@ -709,7 +735,22 @@ fun SaveReminder(
         modifier =
         Modifier
             .clickable {
-                onEvent(ReminderEvent.SaveReminder(title = title, description = description))
+                if (isEventBeingEdited && eventId == "") {
```

Comment:
Instead of making non-existent IDs empty strings, I'd make them `null` instead. That's what every developer understands of a non-existent value. While having a non-nullable string, it's not immediately obvious that this could potentially be empty meaning it's non-existent